install/install.ps1

```diff
@@ -31,7 +31,7 @@ $null = New-Item -ItemType Directory -Force -Path $azureauthDirectory
 
 # Without this, System.Net.WebClient.DownloadFile will fail on a client with TLS 1.0/1.1 disabled
 if ([Net.ServicePointManager]::SecurityProtocol.ToString().Split(',').Trim() -notcontains 'Tls12') {
-    [Net.ServicePointManager]::SecurityProtocol += [Net.SecurityProtocolType]::Tls12
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12;

```

Comment:
Was this semicolon important the whole time? 😱

Reply:
No, not important. The += operator, that's not implemented in PowerShell 4.0.

Reply:
I considered removing this, because it should be needed. In order to download this script itself, you have to have used TLS1.2 - that's why I added it to the README code blocks.

Reply:
Removed.